en/Sorting Algorithms/Merge Sort.md

```diff
@@ -18,9 +18,17 @@ Average - O(n log n)
 Worst case - O(n log n)
 ```
 
+The time complexity can be obtained by solving the below recurrence relation:
+
+$$T(n) = 2T\left(\frac{n}{2}\right) + An$$
+
+
+
 #### Space Complexity
 
-`O(n)`
+```
+O(n)
```

Comment:
Should be `$$O(n)$$`, IMO. Also consider adding an explanation.

Reply:
I changed it to ''' O(n) ''', for consistent display, the time complexity displays in a code block, whereas the space complexity displays as an inline code segment.

![Screenshot_2023-10-18-09-10-27-00_40deb401b9ffe8e1df2f1cc5ba480b12.jpg](https://github.com/TheAlgorithms/Algorithms-Explanation/assets/111554368/f7d94839-b10e-4ef8-ab23-ff73181fce7c)

Reply:
I'll continue with whatever you discern to be better though.

Reply:
I think `$...$` (inline formula) or `$$...$$` is more appropriate for typesetting math. The (ab)use of code blocks and inline code was mostly a workaround for platforms that lack(ed) support for LaTeX, I think.